sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/streaming/ReadLimit.java

```diff
@@ -39,6 +39,8 @@ static ReadLimit minRows(long rows, long maxTriggerDelayMs) {
 
   static ReadLimit maxFiles(int files) { return new ReadMaxFiles(files); }
 
+  static ReadLimit maxBytes(int bytes) { return new ReadMaxBytes(bytes); }
```

Comment:
Ur, why this is `int` instead of `long`? 2GB limit looks too small to me. Is there any reason, @MaxNevermind ?

Reply:
Good catch. `long` looks more proper.

Reply:
fixed